<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="bagPath" default="/home/docker/host_dir/zedPC.bag"/>
    <arg name="outputDir" default="/home/docker/host_dir/rosbagHDFConverter/"/>
    <arg name="topicPointcloud" default="/topic_of_the_pointcloud"/>
    <arg name="iterations" default="50"/>


    <node name="seerep_benchmarking_rosbag_hdf5_converter" pkg="seerep_benchmarking" type="rosbag_hdf5_converter" output="screen">
        <rosparam param="bagPath" subst_value="True">$(arg bagPath)</rosparam>
        <rosparam param="outputDir" subst_value="True">$(arg outputDir)</rosparam>
        <rosparam param="topicPointcloud" subst_value="True">$(arg topicPointcloud)</rosparam>
        <rosparam param="iterations" subst_value="True">$(arg iterations)</rosparam>
    </node>
</launch>
